### PR TITLE
Update QaSchemaIT using TestKit [ECR-3054]

### PR DESCRIPTION
**Pull request by MakarovS**
## Overview
Update QaSchemaIT using TestKit.

---
See: https://jira.bf.local/browse/ECR-3054

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [x] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [x] Public API has Javadoc
- [x] Method preconditions are checked and documented in the Javadoc of the method
- [x] Changelog is updated if needed (in case of notable or breaking changes)
- [x] The continuous integration build passes

### CODE REVIEW

File: exonum-java-binding/qa-service/src/test/java/com/exonum/binding/qaservice/QaSchemaIntegrationTest.java [hunk 1]
```diff
@@ -39,20 +35,19 @@ static void loadLibrary() {
   }
 
   @Test
-  void getStateHashesEmptyDb() throws CloseFailuresException {
-    try (Database db = MemoryDb.newInstance();
-         Cleaner cleaner = new Cleaner()) {
-      Snapshot snapshot = db.createSnapshot(cleaner);
-      QaSchema schema = new QaSchema(snapshot);
+  void getStateHashesEmptyDb() {
+    try (TestKit testKit = TestKit.forService(QaServiceModule.class)) {
```
Comment (bullet-tooth) [inline exonum-java-binding/qa-service/src/test/java/com/exonum/binding/qaservice/QaSchemaIntegrationTest.java]: Do we have a task for implementing testkit extension for junit? 🙃

Comment (dmitry-timofeev) [reply exonum-java-binding/qa-service/src/test/java/com/exonum/binding/qaservice/QaSchemaIntegrationTest.java]: Haven't you been to the grooming?

Comment (MakarovS) [reply exonum-java-binding/qa-service/src/test/java/com/exonum/binding/qaservice/QaSchemaIntegrationTest.java]: > Do we have a task for implementing testkit extension for junit? 🙃

Yes, we do :) ECR-3076 I believe

**Comment by MakarovS** [comment]
> Shall we use testkit here as we don't rely on any of its operations? It helps, however, to get rid of the Cleaner instantiation. Maybe, to make MemoryDb easier to use, we shall add withSnapshot to it, shan't we?

I prefer using TestKit for consistency and replace MemoryDb with it when possible.

**Comment by dmitry-timofeev** [comment]
:ok_man: , though `MemoryDb` is also good as its use doesn't violate local consistency :upside_down_face:

**Pull request merged**